### PR TITLE
BZ 1248095

### PR DESCRIPTION
**Pull request by bfallonf**
Added a note box on maybe having to change the IP address in the insecure-registry option, as well as adding a link to securing the registry, as per BZ:

https://bugzilla.redhat.com/show_bug.cgi?id=1248095

### CODE REVIEW

**Comment by adellape** [comment]
@bfallonf See https://bugzilla.redhat.com/show_bug.cgi?id=1248095#c5

**Comment by bfallonf** [comment]
@adellape Added an Important box about that info. Tried to keep it simple. Let me know if i got anything wrong.

File: admin_guide/install/prerequisites.adoc [hunk 1]
```diff
@@ -316,15 +316,23 @@ OPTIONS='--selinux-enabled --insecure-registry 172.30.0.0/16'
...
 accordingly.
-====
++
+[IMPORTANT]
+172.30.0.0/16 is the default value of the `*servicesSubnet*` variable in the
+*_master-config.yaml_* file. If this has changed, then the `--insecure-registry`
+value in the above step should be adjusted to match, as it is indicating the
+subnet for the registry to use. Note that the `*openshift_master_portal_net*`
+variable can be set in the Ansible inventory file and used during the
+link:advanced_install.html#configuring-ansible.html[advanced installation]
```
Comment (adellape) [inline admin_guide/install/prerequisites.adoc]: Stray `.html` at the end of the anchor link.

**Comment by adellape** [comment]
LGTM other than one minor issue inline.

**Comment by bfallonf** [comment]
@adellape Done. Should be good to merge if there's nothing else.

**Pull request merged**